quickref/qsharp-quick-reference.tex

```diff
@@ -165,22 +175,25 @@
     Call adjoint \newline operation & \texttt{Adjoint \emph{Name}(\emph{parameters})} \\
     Call controlled \newline operation & \texttt{Controlled \emph{Name}(\emph{controlQubits}, \emph{parameters})} \\
   \end{keysref}
-
   \begin{keysref}{Symbols and Variables}
     Declare immutable \newline symbol & \texttt{let \emph{name} = \emph{value}} \\
     Declare mutable \newline symbol (variable)  & \texttt{mutable \emph{name} = \emph{initialValue}} \\
     Update mutable \newline symbol (variable)   & \texttt{set \emph{name} = \emph{newValue}} \\
+    Apply-and-Reassign & \texttt{mutable \emph{name} = \emph{initialValue}} \newline
```

Comment:
This entry needs a general case before we can go into the example (something like "set *name* *operator*= *expression*"), and I don't think we should include a loop in the example - this might be confusing, since one can use apply-and-reassign outside of the loops.